### PR TITLE
vuepress tendermint docs

### PR DESCRIPTION
**Pull request by zramsay**
related to: https://github.com/cosmos/cosmos.network/pull/184

requires: https://github.com/tendermint/tendermint/pull/1763

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Tendermint
+
```
Comment (zramsay) [inline docs/README.md]: this file should go in the tendermint docs

File: .gitmodules [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[submodule "docs/tendermint"]
+	path = docs/tendermint
+	url = https://github.com/tendermint/tendermint
+  branch = develop
```
Comment (jolesbi) [inline .gitmodules]: this is not a big deal - but i feel like master should be for production and develop should be for staging stuff.

**Comment by jolesbi** [comment]
looks good to me

File: .gitmodules (hunk 2 above)
Comment (zramsay) [reply .gitmodules]: agree. we'll switch once the "nice" docs make it to tendermint's master branch

**Comment by zramsay** [comment]
merging to see it it makes it to staging

**Pull request merged**